### ISSUE TITLE
Do NOT reset dates to currentDate() when stages are not yet reached

### ISSUE THREAD

**Issue by frsteinb** [Gremmel/kleiner-brauhelfer#82]
Is there a good reason, why the "Brautag" gets reset to currentDate() when "BierGebraut" is not yet done? I like to plan brews for a specific date and see no good reason for this arbitrary reset. I even hacked a change to MainWindowImpl::setDatumsfelder and MainWindowImpl::on_tabWidged_currentChanged and cannot see any problems. Similar, probably for pitching and bottling.

**Comment by scooterama** [comment]
It used to be the createDate and IMHO it's best to have the current date without worrying when brewing, because in this case the date is set to "now" whenever pressing "Bier gebraut".

**Comment by frsteinb** [comment]
I agree that setting the brew/bottling/emptied date to "now", when clicking the according buttons, makes sense. But this does not mean, that the editable(!) brew date field should get overwritten as long as the button is not clicked (this is what happens now). I see no reason not to use that field to note a future planned brew date.
